db/pf-schema-X.Y.Z.sql

```diff
@@ -122,6 +122,12 @@ INSERT INTO `node_category` (category_id,name,notes) VALUES ("2","guest","Guests
 INSERT INTO `node_category` (category_id,name,notes) VALUES ("3","gaming","Gaming devices");
 
 --
+-- Insert 'voice' category
+--
+
+INSERT INTO `node_category` (category_id,name,notes) VALUES ("4","voice","VoIP devices");
```

Comment:
Why are we specifying an ID ('4' in this case) if the column is auto-increment.
I know that the upgrade schema does not, but unsure why doing it when creating the schema